frappe/desk/doctype/notification_log/notification_log.py

```diff
@@ -124,6 +124,22 @@ def get_email_header(doc):
 	return header_map[doc.type or "Default"]
 
 
+@frappe.whitelist()
+def get_notification_logs(limit=20):
+	notification_logs = frappe.db.get_list(
+		"Notification Log", fields=["*"], limit=limit, order_by="creation desc"
+	)
+
+	users = [log.from_user for log in notification_logs]
+	users = [*set(users)]  # remove duplicates
+	user_info = frappe._dict()
+
+	for user in users:
+		frappe.utils.add_user_info(user, user_info)
```

Comment:
This function doesn't exist on v13. The whole PR isn't required on v13 😅 

Add a test for fixes!